browser/UHR.js

```diff
@@ -109,7 +109,10 @@ UHR.prototype._doRequest = function (parameters) {
 			requestError = new Error(xhr.statusText || ERROR_CONNECTION);
 			reject(requestError);
 		};
-		xhr.onloadend = function () {
+		xhr.onreadystatechange = function () {
+			if (xhr.readyState !== 4) {
+				return;
+			}
```

Comment:
I mean this change could break the existing API in cases when there is no connection or there is a timeout or error. This code just returns in a case when the state is not `4`. I guess, in cases when it's an error it's not `4` and the error won't be handled.